src/email/components/SubHeader.tsx

```diff
@@ -7,28 +7,29 @@ type Props = { children: React.ReactNode };
 
 export const SubHeader = ({ children }: Props) => (
   <>
-    <MjmlSection background-color={background.primary} padding-bottom="0">
+    <MjmlSection
+      backgroundColor={background.primary}
+      padding="24px 12px 0 12px"
+    >
       <MjmlColumn>
-        <MjmlDivider
-          border-width="1px"
-          border-color="#DCDCDC"
-          padding="0 48px"
-        />
+        <MjmlDivider border-width="1px" border-color="#DCDCDC" padding="0" />
       </MjmlColumn>
     </MjmlSection>
-    <MjmlSection background-color={background.primary} padding="0">
+    <MjmlSection
+      backgroundColor={background.primary}
+      padding="0px 12px 20px 12px"
+    >
       <MjmlColumn>
         <MjmlText
-          padding="4px 48px"
-          font-size="20px"
-          line-height="20px"
+          padding="4px 0px"
+          fontSize="20px"
+          lineHeight="23px"
+          letterSpacing="-2%"
```

Comment:
I've not seen this with a negative value before, just curious about what it does :)

Reply:
it reduces the horizontal spacing between the characters https://developer.mozilla.org/en-US/docs/Web/CSS/letter-spacing

Reply:
we use it in this case to as closely as possible match the system default fonts to the guardian fonts, as we can't include guardian fonts in emails

Reply:
makes sense, thanks for the info